code/__DEFINES/is_helpers.dm

```diff
@@ -139,6 +139,9 @@ GLOBAL_LIST_INIT(glass_sheet_types, typecacheof(list(
 #define ispatch(A) istype(A, /obj/item/reagent_containers/patch)
 #define isfood(A) istype(A, /obj/item/food)
 
+GLOBAL_DATUM_INIT(regex_rgb_text, /regex, regex(@"^#?(([0-9a-fA-F]{8})|([0-9a-fA-F]{6})|([0-9a-fA-F]{3}))$"))
```

Comment:
Could you put this next to GLOB.is_color. We want both of these, because `is_color` should be restricted to 6 digits, while its fine if this one picks up 3, 6, and 8